### PR TITLE
[docs] update dev client screenshot

### PR DESCRIPTION
**Pull request by tcdavis**
# Why

They are quite out of date.

# How

Took new screenshots and resized / optimized

# Test Plan

Ran locally, image looks fine
image

### CODE REVIEW

**Review by lukmccall — APPROVED** [review]

**Review by brentvatne — APPROVED** [review]

**Review by esamelson — APPROVED** [review]
LGTM! we should update again once #17633 lands 🙂

**Review by ajsmth — APPROVED** [review]

**Pull request merged**
